apps/browser/qml/pages/components/BrowserListItem.qml

```diff
@@ -0,0 +1,54 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+ListItem {
+    id: root
+
+    property alias label: valueButton.label
+    property alias value: valueButton.value
+    property alias iconSource: icon.source
+    property alias description: valueButton.description
+
+    width: parent.width
+    contentHeight: Math.max(Theme.itemSizeMedium, row.height)
+    onClicked: openMenu()
+
+    Row {
+        id: row
+
+        width: parent.width - 2*Theme.horizontalPageMargin
+        x: Theme.horizontalPageMargin
+        anchors.verticalCenter: parent.verticalCenter
+
+        Icon {
+            id: icon
+
+            anchors.verticalCenter: parent.verticalCenter
```

Comment:
The "value" text of `valueButton` is animated nicely if it moves onto a new line, whereas the icon's vertical position will jump without any animation, which looks jarring. It would be good to add a behavior to the icon's `y` value, so that it animates nicely. Something like the following will match the animation of ValueButton:

`Behavior on y { NumberAnimation{ easing.type: Easing.InOutQuad; duration: valueButton._duration } }`

Note that you'll need to set `y` explicitly for this to work, rather than setting the `verticalCenter`.

Reply:
what should be` y`? `Y` position is not clear to me .. can it be calculated correctly

Reply:
Currently the verticalCenter value is equivalent to this, I think:
```
y: valueButton.y + (valueButton.height + height) / 2
```